ImporterContext.hpp

```diff
@@ -52,11 +52,14 @@ class ImporterContext final : public IImporterContext {
   }
   virtual nvinfer1::IPluginLayer* addPlugin(Plugin* plugin,
                                             std::vector<nvinfer1::ITensor*> const& inputs) override {
-    _owned_plugin_instances.emplace_back(plugin);
+    // Note: Plugins are wrapped here to make them work with
+    // onnx2trt::PluginFactory.
+    auto* wrapped_plugin = new TypeSerializingPlugin(plugin);
```

Comment:
Who is going to destruct `wrapped_plugin`?

Reply:
It's added to `_owned_plugin_instances` and has the lifetime of the ImporterContext object.